plutus-contract/src/Plutus/Contract/Request.hs

```diff
@@ -150,7 +150,11 @@ waitNSlots n = do
   c <- currentSlot
   awaitSlot $ c + fromIntegral n
 
--- | Wait until the given time.
+-- | Wait until the slot where the given time falls into and returns the end
+-- time of the slot.
```

Comment:
For this and the one below, I wonder if it's clearer to say we return "the latest time that we know has passed" or something? That happens to be the end of the slot, but imagine that for some reason we had to go on two slots... I think we'd want to return the end of the second slot, because that's how far we've got.